docs/runtime.android.md

```diff
@@ -1,121 +1,110 @@
-## ~`getRuntimeConfiguration()`~
+## String replacement
+On Android, the only string replacement needed is the platform name. We substitute this:
```

Comment:
Notable change here. Moved the runtime config to `getRuntimeConfiguration` instead of string replacement. String replacement now it's extremely basic.